### PR TITLE
fix: Date Selector not working on Highlight Creation

### LINKED ISSUE #1756: Bug: Date Selector not working on Highlight Creation/Editing Dialogs
### Describe the bug

While attempting to create a new highlight on OpenSauced Insights today, I encountered an issue with the date selector. I found that I am unable to use the date selector in both the highlight creation and editing dialogs. Here are my observations:

1. Clicking on a part of the date selector popup within the parent dialog body results in the date selector closing without selecting a date.
2. Clicking on a part of the date selector popup that extends outside of the parent dialog body (i.e., the dialog background overlay that closes the dialog when clicked) causes the entire highlight creation/editing dialog to close without selecting the date.

### My Analysis

Based on the above observations, I believe the issue is related to the position of the date selector on the z-axis, specifically its `z-index` or `position` property. It appears that the popup element visually sits on top of the dialog but is not actively responsive to user interactions.

### Proposed Solutions:

In such cases, an element exhibiting this behavior, like the date selector popup, may require a simple adjustment in its `position` property (e.g., setting it to `relative`, `absolute`, etc.). This adjustment can complement the existing `z-index` value, ensuring that the date selector is not only visually on top but also fully interactive. To resolve this issue, I suggest investigating and adjusting the `position` property as a potential solution. Some CSS debugging may be needed to fine-tune this behavior.

#### Highlight Creation

[screencast-insights.opensauced.pizza-2023.09.27-09_03_54 (1).webm](https://github.com/open-sauced/app/assets/25631971/d51916af-86d6-4a6b-ba5d-9b5dfffb15e4)

#### Highlight Editing

[screencast-insights.opensauced.pizza-2023.09.27-09_03_54.webm](https://github.com/open-sauced/app/assets/25631971/97998421-1f66-42d4-95d8-8e405ee3bf85)

### Browsers

Chrome

### Additional context (Is this in dev or production?)

Production

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by aryanas159**
…iting Dialogs

fixed issue #1756: Date Selector not working on Highlight Creation/Editing Dialogs

## Description
added pointer-events-auto to PopoverContent's class names.

fixes #1756 
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recording

https://github.com/open-sauced/app/assets/114330931/57aca967-d6e5-4803-8611-add6f1f6b1d7

https://github.com/open-sauced/app/assets/114330931/1149b9ff-3d0d-47e1-a376-65a2855fc6b5

s

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/molecules/ContributorHighlight/contributor-highlight-card.tsx [hunk 1]
```diff
@@ -652,7 +652,7 @@ const ContributorHighlightCard = ({
                             {date && <span className="text-xs">{format(date, "PPP")}</span>}
                           </button>
                         </PopoverTrigger>
-                        <PopoverContent className="w-auto p-0 bg-white">
+                        <PopoverContent className="w-auto p-0 bg-white pointer-events-auto">
```
Comment (RitaDee) [inline components/molecules/ContributorHighlight/contributor-highlight-card.tsx:655]: LGTM 🍕

**Comment by babblebey** [comment]
Hey @aryanas159

Remove the `issue***:` from the PR title, leaving only the `fix: Bug******`.

This will let you satisfy PR compliance.

**Comment by aryanas159** [comment]
> Hey @aryanas159
> 
> Kindly remove the `issue 1756:` from the PR title, leaving only the `fix: Bug******`.
> 
> This will let you satisfy PR compliance.

Thanks for the heads up @babblebey 
I have changed the title.

**Comment by aryanas159** [comment]
> For ease of review next time @aryanas159, considering this PR fixes something very visual, you might want to include screencasts (i.e. picture or video) to demo the changes you made.
> 
> I would've loved to see a video demo for this one though, do you think you can get that!? 😉
> 
> Add it right under the `Mobile & Desktop Screenshots/Recordings` section on the PR Description template 😉

@babblebey  I though I had added the screencasts, but for some reason they weren't uploaded, I have added them again.

**Comment by RitaDee** [comment]
> > For ease of review next time @aryanas159, considering this PR fixes something very visual, you might want to include screencasts (i.e. picture or video) to demo the changes you made.
> > I would've loved to see a video demo for this one though, do you think you can get that!? 😉
> > Add it right under the `Mobile & Desktop Screenshots/Recordings` section on the PR Description template 😉
> 
> @babblebey I thought I had added the screencasts, but for some reason they weren't uploaded, I have added them again.

I can see it on my end.

**Review by babblebey — COMMENTED** [review]
🍕

**Review by OgDev-01 — APPROVED** [review]
LG\TM

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**